src/styles/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
```

Comment:
card should have padding: 32px 16px 16px; to avoid adding margin-left/margin-right for inner elements